app/grants/models.py

```diff
@@ -1087,8 +1119,18 @@ def successful_contribution(self, tx_id):
 
 @receiver(pre_save, sender=Grant, dispatch_uid="psave_grant")
 def psave_grant(sender, instance, **kwargs):
+    # TODO: move to grant task
+    instance.clr_prediction_curve = instance.calc_clr_prediction_curve
+    instance.clr_round_num = instance.calc_clr_round_nums
+
+    # todo: move to grant task
+    active_clr_rounds = GrantCLR.objects.filter(is_active=True)
```

Comment:
@thelostone-mc theres probably a smarter way to manage the `instance.active_clr_rounds` object i have not found it yet. kind of feels like it should be a computed property & not a fk to me, but open to feedback of course.

Reply:
@owocki I think this can be removed 
This function does the same 
https://github.com/gitcoinco/web/blob/master/app/grants/views.py#L1312

Reply:
per our slack convo; we are keeping it right? so that if a grant is updated its fk's are updated